plugin/build.gradle

```diff
@@ -101,7 +101,7 @@ dependencies {
     // Plugins to be available for single tests using the @WithPlugin annotation.
     jenkinsPlugins 'org.jenkins-ci.plugins:cloudbees-folder:6.1.0'
     jenkinsPlugins 'org.jenkins-ci.plugins:gradle:1.22' // Old version used to test reporting of outdated plugins.
-    jenkinsPlugins 'org.jenkins-ci.plugins:htmlpublisher:1.12' // Used to test reporting of deprecated plugins.
+    jenkinsPlugins 'org.jenkins-ci.plugins:gradle:1.26' // Old version used to test reporting of deprecated plugins.
```

Comment:
Is it worth mentioning in the commit message why this was also changed from `htmlpublisher`?

Reply:
@mnonnenmacher did you forget about this one?

Reply:
Actually I missed it. Done.